src/utilities/util_dump.c

```diff
@@ -161,17 +159,6 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
         }
         if ((p = strchr(simpleuri, '(')) != NULL)
             *p = '\0';
-        /*
-         * If we're dumping the history store, we need to set this flag to ignore tombstones. Every
-         * record in the history store is succeeded by a tombstone so we need to do this otherwise
-         * nothing will be visible. The only exception is if we've supplied a timestamp in which
-         * case, we're specifically interested in what is visible at a given read timestamp.
-         */
-        if (WT_STREQ(simpleuri, WT_HS_URI) && timestamp == NULL) {
```

Comment:
Doesn't your change here break the ability to dump the history store as of a timestamp?

Reply:
oh, that's correct. I'll revert it.